src/gui/search/searchjobwidget.cpp

```diff
@@ -198,8 +201,10 @@ void SearchJobWidget::cancelSearch()
 void SearchJobWidget::downloadTorrents()
 {
     const QModelIndexList rows {m_ui->resultsBrowser->selectionModel()->selectedRows()};
-    for (const QModelIndex &rowIndex : rows)
+    for (const QModelIndex &rowIndex : rows) {
+        setRowColor(rowIndex.row(), QApplication::palette().color(QPalette::LinkVisited));
```

Comment:
Duplicates code in `SearchJobWidget::onItemDoubleClicked()`. I would put that into `SearchJobWidget::downloadTorrent()`.

Reply:
Done.